gradle/dependency-locking.gradle

```diff
@@ -0,0 +1,19 @@
+// see https://docs.gradle.org/current/userguide/dependency_locking.html
```

Comment:
I have not seen a gradle lockfile before 🤦 . Very nice - we are stealing for Workbench..